src/Uno.UI/UI/Xaml/UIElement.mux.cs

```diff
@@ -858,6 +858,15 @@ internal virtual void Enter(EnterParams @params, int depth)
 
 		internal virtual void Leave(LeaveParams @params)
 		{
+			if (__Store.TryGetOnDemandProperty(KeyboardAcceleratorsProperty, out var acceleratorCollection))
+			{
+				// HACK: We should generalize Leave to leave all relevant DPs in a general way.
+				// Relevant WinUI pieces:
+				// https://github.com/microsoft/microsoft-ui-xaml/blob/539e4de5bb6bf5973cc74110aa926b450b8aa53c/dxaml/xcp/core/core/elements/depends.cpp#L1271-L1290
+				// https://github.com/microsoft/microsoft-ui-xaml/blob/539e4de5bb6bf5973cc74110aa926b450b8aa53c/dxaml/xcp/components/DependencyObject/PropertySystem.cpp#L1405-L1428
+				(acceleratorCollection as KeyboardAcceleratorCollection)?.Leave(@params);
```

Comment:
This is currently only for `UNO_HAS_ENHANCED_LIFECYCLE`.

@MartinZikmund Would you be able to adjust for other platforms?

The Enter should hopefully be working for all platforms, as it's coming from DependencyObjectStore call to `EnterEffectiveValue`

Reply:
`Enter` is currently also problematic for other platforms.